core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to show a list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id')
+        exploration = exp_fetchers.get_exploration_by_id(exploration_id)
```

Comment:
Validate `exploration_id` is not None.

Reply:
it will be validated from frontend i.e we can't send empty exploration id in this method

Reply:
Don't rely on frontend validation assume the handler to be a separate service (as an API).

Reply:
Done

Reply:
@aggarwalpulkit596, is this resolved?

Reply:
yes